derp/derp_server.go

```diff
@@ -1323,6 +1329,21 @@ func (s *Server) ExpVar() expvar.Var {
 	m.Set("average_queue_duration_ms", expvar.Func(func() interface{} {
 		return math.Float64frombits(atomic.LoadUint64(s.avgQueueDuration))
 	}))
+	m.Set("top_100_bandwidth_ips", expvar.Func(func() interface{} {
```

Comment:
I think this is going to be too expensive to do on every metrics scrape.

I think we need a per-client counter and also a global low watermark of what score would put us in the top 100, and then do an O(1) addition to/from the leaderboard set as needed, without O(n) work like this